packages/shared/src/components/cards/AdLink.tsx

```diff
@@ -15,7 +15,7 @@ export default function AdLink({ ad, onLinkClick }: AdLinkProps): ReactElement {
       rel="noopener"
       title={ad.description}
       onClick={() => onLinkClick?.(ad)}
-      onMouseUp={(event) => event.button === 1 && onLinkClick?.(ad)}
+      onAuxClick={(event) => event.button === 1 && onLinkClick?.(ad)}
```

Comment:
FYI auxclick is not [supported on Safari](https://caniuse.com/auxclick) 😬

Reply:
Weird, I tested it now and it fires correctly on Safari 🤯
I'll try mobile to check as well

Reply:
What is the intended action to happen?

Reply:
Best example:
Card -> Read article button
Middle click it
Actions to happen:
- Open in new tab
- Track event for go to link

Reply:
Never mind my mobile comment lol 😂

Reply:
Well, with the new iPhone 17 with USB-C you can connect a mouse I think 🤓

Reply:
My local environment has crashed hard right now, so I can't test locally.
Though I think the default action for middle click on macOS is to open links in new tab, probably why it works.

You could try to remove `onAuxClick` and test it in Safari and see if it still works

Reply:
Yep you are totally right, removing onAuxClick will still work on Safari indeed.
It opens new tab + tracks the link

Chrome with it disabled:
Opens new tab + does NOT track the link

With auxClick they both work as intended

Reply:
Perfect 😄

Reply:
> My local environment has crashed hard right now, so I can't test locally.
Though I think the default action for middle click on macOS is to open links in new tab, probably why it works.

> You could try to remove onAuxClick and test it in Safari and see if it still works

Good to know, I know we tested `onAuxClick` on firefox and safari and it worked, but did not know about default behaviour 🤘